We have long reads and short read transcriptomes.  With these enormous amount of data, we did the following:  

[De novo assembly of long read transcriptome](https://gitlab.com/douglas-fir-transcriptome/de-novo-assembly-of-long-reads)
*   [Quality control of isoseq subreads](https://gitlab.com/douglas-fir-transcriptome/de-novo-assembly-of-long-reads/tree/Isoseq3-quality-control)
*   [Assembly of reference transcriptome](https://gitlab.com/douglas-fir-transcriptome/de-novo-assembly-of-long-reads/tree/Reference-transcriptome)  
*   [Identify transcription factors](https://gitlab.com/douglas-fir-transcriptome/de-novo-assembly-of-long-reads/tree/Identify-TF-using-Entap-annotation)
*   [Predict transcription factors and domains](https://gitlab.com/douglas-fir-transcriptome/de-novo-assembly-of-long-reads/tree/Identify-TF)  
*   [Prediction of long non-coding RNA](https://gitlab.com/douglas-fir-transcriptome/de-novo-assembly-of-long-reads/tree/Predict-lncRNA)  
*   [Identify miRNA](https://gitlab.com/douglas-fir-transcriptome/de-novo-assembly-of-long-reads/tree/Identify-miRNA)  

[Genome guided assembly of long read transcriptome](https://gitlab.com/douglas-fir-transcriptome/Genome-guided-assembly-of-long-reads)  

[Coding genome reconstruction](https://gitlab.com/douglas-fir-transcriptome/Coding-genome-reconstruction-using-long-reads)  

[De novo assembly of short reads](https://gitlab.com/douglas-fir-transcriptome/De-novo-assembly-of-short-reads)    

[Hybrid sequencing](https://gitlab.com/douglas-fir-transcriptome/hybrid-sequencing)  